config-provisioning/src/main/java/com/yahoo/config/provision/NodeType.java

```diff
@@ -14,6 +14,9 @@
     /** Nodes running the shared proxy layer */
     proxy(false),
 
+    /** A host of a (docker) proxy node */
+    proxyhost(true),
```

Comment:
There are a couple of switch-statements that seems to handle most/all types and that you may want to update at the same time?